src/app/contact/contact.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-contact',
+  templateUrl: './contact.component.html',
+  styleUrls: ['./contact.component.scss']
+})
+export class ContactComponent implements OnInit {
+
+  constructor() 
+  { 
+    // (document).ready(function() {
+    //   $("#linkedin-nadia-button").click(function() 
+    //   {
+    //       window.location = "https://www.linkedin.com/in/nadianoguesalmeida/";
+    //   });
+    // });
+
+  }
+
+  redirectLinkedinNadia($event){    
+    console.log("Save button is clicked!", $event);    
```

Comment:
Do we really need to log this message here?

Reply:
No